crates/milli/src/search/new/geo_sort.rs

```diff
@@ -73,6 +71,26 @@ impl Strategy {
     }
 }
 
+#[cfg(not(test))]
+fn default_max_bucket_size() -> u64 {
+    1000
+}
+
+#[cfg(test)]
+static DEFAULT_MAX_BUCKET_SIZE: std::sync::Mutex<u64> = std::sync::Mutex::new(1000);
+
+#[cfg(test)]
+pub fn set_default_max_bucket_size(n: u64) {
+    let mut size = DEFAULT_MAX_BUCKET_SIZE.lock().unwrap();
+    *size = n;
+}
+
+#[cfg(test)]
+fn default_max_bucket_size() -> u64 {
+    let max_size = *(DEFAULT_MAX_BUCKET_SIZE.lock().unwrap());
+    max_size
+}
```

Comment:
Hey, sorry but since multiple tests are run in parallel sharing the same variable this will be super flaky.
One test may change the default max bucket size for the other 😭 

Maybe the easiest thing is to just shove the value into the `Strategy` like we did for the cache size.
Either you just add it to every variant, or you could create a structure `Parameter` that contains both the max_bucket_size and the cache_size + a new unit enum `Strategy` that doesn’t hold any data.

When calling this value from Meilisearch we always set it to `default` iirc, so you can do whatever you prefer it’s only local to this file.

Reply:
😂 I think I would prefer to add a Parameter to hold all the relevant parameters